sql/2021/ecommerce/median_lighthouse_score_ecommsites.sql

```diff
@@ -0,0 +1,25 @@
+#standardSQL
+# 13_20: Lighthouse category scores per eCommerce plaforms. Web Almanac run LightHouse only in mobile mode and hence references to mobile tables
+# See https://github.com/HTTPArchive/almanac.httparchive.org/pull/1087#issuecomment-684983795. Due to this, using Sep-2020 in this query
```

Comment:
This comment is obsolete
```suggestion
```

Reply:
Will fix this as part of the above query